resources/views/partials/navbar/menu-item.blade.php

```diff
@@ -1,10 +1,14 @@
 @inject('menuItemHelper', \JeroenNoten\LaravelAdminLte\Helpers\MenuItemHelper)
 
-@if ($menuItemHelper->isSearchBar($item))
+@if ($menuItemHelper->isSearchBar($item) && !$item['topnav'])
```

Comment:
This is not good, if you configure `topnav_right=true` this will thrown an error. The logic on this file should be keep as it was.